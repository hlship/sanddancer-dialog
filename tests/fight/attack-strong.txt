(setup coyotes offer)
(now) (knock holds #strength)
take jade
w
(now) (turns in current room 6)
w
attack coyotes
