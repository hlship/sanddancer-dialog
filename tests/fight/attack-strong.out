

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> (setup chase)
Query succeeded: (setup chase)
> (now) (knock holds #strength)
> take jade
You take the piece of jade.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> n
(first attempting to get out of the pickup truck)
The door creaks open with difficulty, rusted joints straining. You get out of
the pickup truck.

You walk north.

Crumbling Concrete
This building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

> w
You walk west for a long time, thoughts turning over like the sand under your
sneakers. You think about your shit job. You think about Ocean and how pissed
she's gonna be that you're not home. Like you tell her not to wait up for you
but she always does anyway.

Some wind kicks up and chucks sand in your face. You blink it out and kinda
realize while you're doing it that you haven't really been paying attention to
where you've been going. Another gust whines through the sagebrush and you
realize with a kind of heavy feeling that with all the dust this wind is kicking
up you can't see a damn thing. You can't see your pickup, you can't see the
electrical tower. Not even your damn footprints. You're lost. Well, shit.

Open Desert
Clouds of dust swirl through the air, and you shiver in the chill night air.

All you can make out in the darkness are a dead cow, a looming cactus, and a
dull grey boulder.

> (now) (#chase has completed)
> z
Nah, man, better keep busy or else it's back to brooding about your grandma's
stories.

It's over. They've caught you. They surround you, circling, growling, looking
for weakness, moving in for the kill.

> attack coyotes
It's over. The coyotes dive in for the kill, jaws snapping, and without hardly
realizing what you're doing you make a fist and slam it into the one that's
closest.

The coyote squeals, flung sideways by the strength of your punch, blood dripping
from its nose, and the rest of the pack pulls back in sudden caution. Another
coyote makes to lunge and you whirl towards it in fury, pulling back for another
blow, but the pack breaks in some collective decision and scatters, mangy
shadows slinking away into the desert. Except for one last coyote who stands his
ground.

And then you blink, and realize he's not a coyote after all. Just... just a guy.

He wears a couple of days of stubble, a scuffed and faded denim jacket, and,
ridiculously in the darkness, a pair of cheap sunglasses. He lights a cigarette,
then holds up a hand in annoyance to block your flashlight beam. "Turn that
thing off," he says, and either you do it or it goes off by itself, because the
next thing you know, it's darker and he's closer and the only light comes from
the red glow of the cigarette.

"That's better," he says. He takes a drag on the cigarette, then pulls another
from behind his ear and offers it to you. "Smoke?"

You could say yes or say no.

> 
